src/server/accessors/Notifier.ts

```diff
@@ -30,6 +35,26 @@ export class Notifier implements INotifier {
         await this.msgBridge.doNotifyRoom(room, message, this.appId);
     }
 
+    /**
+     * Sends a direct message to a user.
+     *
+     * @param {IUser} user - The user to send the message to.
+     * @param {IMessage} partialMsg - The partial message to send.
+     * @returns {Promise<void>} A Promise that resolves when the message has been sent.
+     */
+    public async notifyDM(user: IUser, partialMsg: Omit<IMessage, 'room'>): Promise<void> {
+        const sender = partialMsg.sender || (await this.userBridge.doGetAppUser(this.appId));
```

Comment:
Do we anywhere else hve this default behaviour of falling back to app user? if not this would be inconsistent and confusing IMO. In that case method should fail if sender is not passed (iirc sender is mandatory in Imessage)